content/en/docs/components/pipelines/v1/reference/api/kubeflow-pipeline-api-spec.html

```diff
@@ -1,8 +1,3 @@
-+++
```

Comment:
Is the deletion here correct? I thought the metadata header has some effect to the website layout or display.